src/SFA.DAS.EmployerFinance/Configuration/AzureTableStorage/AzureTableStorageConfigurationProvider.cs

```diff
@@ -0,0 +1,117 @@
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace SFA.DAS.EmployerFinance.Configuration.AzureTableStorage
+{
+    //todo: this code will be replaced by the nuget package generated from this PR, once the package is ready... (https://github.com/SkillsFundingAgency/das-shared-packages/pull/281)
+    //todo: implement reload on change if table supports it?
+    //todo: use new table code in cosmos package ?? not currently an option: https://github.com/Azure/azure-cosmos-dotnet-v2/issues/344
+    // ^^ could use this preview package: https://www.nuget.org/packages/Microsoft.Azure.Cosmos.Table/0.10.1-preview
+    public class AzureTableStorageConfigurationProvider : ConfigurationProvider
```

Comment:
This class and others related will be replace by the nuget package generated from the [new shared config PR](https://github.com/SkillsFundingAgency/das-shared-packages/pull/281), so please add any comments about the config helpers to that PR 🙂.